DockerfileDevDocs

```diff
@@ -0,0 +1,29 @@
+FROM amsterdam/python:3.8-buster AS builder
+
+RUN apt-get update \
+ && apt-get install -y --no-install-recommends \
+      graphviz \
+      fonts-liberation \
+      make \
+      git \
+ && rm -rf /var/lib/apt/lists/*
+
+WORKDIR /docs
+COPY dev-docs/requirements.txt /docs
+RUN pip install -r requirements.txt
+
+COPY dev-docs/.gitconfig /root/
```

Comment:
I get no such file or directory for .gitconfig (forgot to add that file?).

Reply:
ouch.